src/swapd/runtime.rs

```diff
@@ -795,7 +795,7 @@ impl SyncerState {
     }
     fn watch_addr_btc(&mut self, script_pubkey: Script, tx_label: TxLabel) -> Task {
         let id = self.tasks.new_taskid();
-        let from_height = self.height(Coin::Bitcoin);
+        let from_height = self.height(Coin::Bitcoin) - 5;
```

Comment:
This can underflow, maybe do `cmp::min(self.height(Coin::Bitcoin) - 5, self.height(Coin::Bitcoin))`

Reply:
true

Reply:
c95e90e